src/components/WidgetText/WidgetTextPresentation.js

```diff
@@ -30,6 +30,8 @@ function WidgetTextPresentation({ widgetData }) {
   } = widgetData.settings
   const format = widgetData?.format || FORMAT_LEFT
 
+  console.log(url)
```

Comment:
This can be removed. I'll make a note to verify that es-lint config is set up properly as this should have been flagged on your local and not been allowed to commit. 
The other case would be maybe the pre-commit actions did not run properly, for which I am available for a call and help figure out any issues related to that.